health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PlanInbox.js

```
@@ -493,24 +502,29 @@ const PlanInbox = () => {
-      {isRootApprover && isStatusConditionMet(activeFilter) &&
+      {isRootApprover && isStatusConditionMet(activeFilter) && (
         <ActionBar
           actionFields={[
-            <Button icon="CheckCircle" label={t(`HCM_MICROPLAN_FINALIZE_MICROPLAN`)} onClick={handleActionBarClick} type="button" variation="primary" />,
+            <Button
+              icon="CheckCircle"
+              label={t(`HCM_MICROPLAN_FINALIZE_MICROPLAN`)}
+              onClick={handleActionBarClick}
+              type="button"
+              variation="primary"
+            />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)